services/notifications/pkg/email/templates/spaces/unsharedSpace.email.body.tmpl

```diff
@@ -0,0 +1,18 @@
+Hello {{ .SpaceGrantee }},
+
+{{ .SpaceSharer }} has removed you from "{{ .SpaceName }}".
+
+You might still have access through your other groups or direct membership. Click here to check it: {{ .ShareLink }}
+
+----------------------------------------------------------
+
+Hallo {{ .SpaceGrantee }},
+
+{{ .SpaceSharer }} hat dich aus dem Space "{{ .SpaceName }}" entfernt.
+
+Du könntest über deine anderen Gruppen oder direkten Mitgliedschaft noch Zugriff haben. Klicke hier zum Überprüfen: {{ .ShareLink }}
```

Comment:
Shouldn't that be 
```suggestion
Du könntest über deine anderen Gruppen oder direkte Mitgliedschaft noch Zugriff haben. Klicke hier zum Überprüfen: {{ .ShareLink }}
```

Reply:
I'm not sure either but my thought was this '[...] Gruppen oder (deiner) direkten Mitgliedschaft [...]'. And in that case I think there needs to be an 'n'. https://de.wiktionary.org/wiki/Flexion:direkt

Reply:
OK. If you say so I'm fine. I just stumbled over it. Maybe add another `deiner`? But as said I'm from Bavaria, I have no clue about proper German :smile:

Reply:
Ok, I'll add another 'deiner' and in the future someone with more knowledge can change it again if it is still wrong.